.github/workflows/build.yml

```
@@ -48,35 +48,36 @@ jobs:
+        uses: actions/setup-dotnet@v4
         with:
           dotnet-version: 3.1.x
 
       # Building requires an up-to-date .NET SDK.
 
       - name: Install .NET 6.0
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 6.0.x
+         dotnet-version: 6.0.x
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 9

(indentation)